browser-client.cpp

```diff
@@ -168,7 +169,7 @@ bool BrowserClient::OnProcessMessageReceived(
 		CefProcessMessage::Create("executeCallback");
 
 	CefRefPtr<CefListValue> args = msg->GetArgumentList();
-	args->SetInt(0, message->GetArgumentList()->GetInt(0));
+	args->SetInt(0, arguments->GetInt(0));
```

Comment:
I think it would probably be more ideal to rename these two variables, `arguments` and `args`, to something like `input_args` and `execute_args`. Having two variables in the same function with nearly identical names but different purposes is confusing to the reader.